pulsar-jms/src/main/java/com/datastax/oss/pulsar/jms/PulsarConnection.java

```diff
@@ -996,6 +988,14 @@ private ConnectionConsumer buildConnectionConsumer(
     return connectionConsumer;
   }
 
+  private void createNonPartitionedTopic(String name) {
+    try {
+      factory.getPulsarAdmin().topics().createNonPartitionedTopic(name);
+    } catch (Exception err) {
```

Comment:
Please catch the specific exception that you expect, otherwise even network errors will be ignored

Reply:
Implemented